### ISSUE TITLE
GroupMilestone 4

### ISSUE THREAD

**Issue by lanista38** [BeanCartel/CoffeeSpace#4]
The actual code is in the sideBranch but the readme is on both.

/cc @codepathreview

**Comment by hatu87** [comment]
You must split works between students by using Github issues.

**Comment by Supremekhaoz** [comment]
We individually have to create issues for the part we worked on?

**Comment by hatu87** [comment]
Yes, you must create works with github issues and assign them to other students.
